lib/miq_expression.rb

```diff
@@ -96,13 +97,13 @@ def self._to_human(exp, options = {})
     when "between dates", "between times"
       col_name = exp[operator]["field"]
       col_type = get_col_type(col_name)
-      col_human, dumy = operands2humanvalue(exp[operator], options)
+      col_human, = operands2humanvalue(exp[operator], options)
```

Comment:
WDYT about global disabling the rule that complains about your preferred version?